### DISCUSSION TITLE
YouTube attempting to shut down 3rd-party clients that do not show ads

### DISCUSSION THREAD

**Discussion by mcmurphy8097234789** [TeamNewPipe/NewPipe#10981] [Questions] [answered]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my question isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

https://arstechnica.com/gadgets/2024/04/youtube-will-start-blocking-third-party-clients-that-dont-show-ads/?comments=1&comments-page=1

Title basically. Sounds like youtube is going on a new crusade. Just wondering, how NewPipe plans on dealing with this. Thoughts?

**Comment by shuvashish76** [comment, marked as answer]
Duplicate
https://github.com/TeamNewPipe/NewPipe/discussions/10979#discussioncomment-9133579
